code/modules/mob/living/living_emote.dm

```diff
@@ -98,7 +99,10 @@
 		var/mob/living/carbon/human/H = user
 		if(H.dna.species)
 			message = H.dna.species.death_message
-			return pick(H.dna.species.death_sounds)
+			if(H.is_muzzled())
+				return
```

Comment:
I'd personally recommend moving this out to an override of `should_play_sound()` instead of having it in `get_sound()`.

Reply:
This helped me get it to work properly as intended! Cheers for the help Lewc ^^